guild/plugins/python_script.py

```diff
@@ -243,12 +243,13 @@ def _cache_valid(cache_path, mod_path):
     def _load_and_cache_flags_data(
         self, mod_path, mod_package, sys_path, base_args, flags_dest, cached_data_path
     ):
-        if (
-            os.getenv("NO_IMPORT_FLAGS_PROGRESS") != "1"
-            and os.getenv("FLAGS_TEST") != "1"
-            and not os.getenv("_GUILD_COMPLETE")
-        ):
-            cli.note_once("Refreshing flags...")
+        # if (
+        #     os.getenv("NO_IMPORT_FLAGS_PROGRESS") != "1"
+        #     and os.getenv("FLAGS_TEST") != "1"
+        #     and not os.getenv("_GUILD_COMPLETE")
+        #     and not os.getenv("SUPPRESS_REFRESHING_FLAGS")
+        # ):
+        #     cli.note_once("Refreshing flags...")
```

Comment:
This was showing up across many tests unexpectedly, I think because of the separate processes being used for parallel execution. In hindsight, maybe an ellipsis to catch this makes more sense.